client/app/components/template-explorer/template-explorer.component.js

```diff
@@ -0,0 +1,204 @@
+/* eslint camelcase: "off" */
```

Comment:
Every single component we have uses this existing syntax.  Angular has its own mechanism for loading up templates.  Not sure we want to bypass it .